pyproject.toml

```diff
@@ -50,8 +50,8 @@ developer = [
 doc = [
     'numpy>=1.22',
     'matplotlib>=3.5',
-    'pydata-sphinx-theme>=0.13',
-    'sphinx>=6',
+    'pydata-sphinx-theme>=0.13.3',
+    'sphinx>=7',
```

Comment:
this might need a `<7.2` until pydata-sphinx-theme releases a new version, see https://github.com/pydata/pydata-sphinx-theme/issues/1404 (which could have at one point read "incompatibility with Sphinx 7.2")... but they just cut 7.2.1 so we'll see!

Reply:
7.2.1 worked for me on my computer...